consensus/ibft/fork/fork.go

```diff
@@ -0,0 +1,157 @@
+package fork
+
+import (
+	"encoding/json"
+	"errors"
+
+	"github.com/0xPolygon/polygon-edge/helper/common"
+	"github.com/0xPolygon/polygon-edge/validators"
+)
+
+const (
+	// Keys in IBFT Configuration
+	KeyType          = "type"
+	KeyTypes         = "types"
+	KeyValidatorType = "validator_type"
+)
+
+var (
+	ErrUndefinedIBFTConfig = errors.New("IBFT config is not defined")
+)
+
+// IBFT Fork represents setting in params.engine.ibft of genesis.json
+type IBFTFork struct {
+	Type          IBFTType                 `json:"type"`
+	ValidatorType validators.ValidatorType `json:"validator_type"`
+	Deployment    *common.JSONNumber       `json:"deployment,omitempty"`
+	From          common.JSONNumber        `json:"from"`
+	To            *common.JSONNumber       `json:"to,omitempty"`
+
+	// PoA
+	Validators validators.Validators `json:"validators"`
```

Comment:
should thid be omitempty

Reply:
Added 422a66c838677bcd6df5c82fc9fe0a8a4d904974